plugins/swf-backend/src/service/JiraService.ts

```diff
@@ -36,6 +36,16 @@ export interface IssueCommented extends BaseIssueEvent {
   };
 }
 
+export interface IssueResolved extends BaseIssueEvent {
+  issue_event_type_name: 'issue_resolved';
+  changelog: {
+    items: {
+      field: string;
+      fromString: string;
+      toString: string;
+    }[];
+  };
+}
 export interface IssueUpdated extends BaseIssueEvent {
   issue_event_type_name: 'issue_generic';
```

Comment:
Instead of creating a new type `IssueResolved`, you can update this property like this:
```suggestion
  issue_event_type_name: 'issue_generic' | 'issue_resolved';
```

Reply:
👍